backpack-compose/build.gradle

```diff
@@ -0,0 +1,80 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'com.android.library'
```

Comment:
i've noticed that the other gradle file can include the token files without it being in kotlin, so renamed this one as well as the syntax wasn't ideal with our current setup and it was inconsistent with other modules. no actual changes here

Reply:
I would keep it as it is and migrated the other Gradle files to kts instead (in another ticket). Working with Kotlin is easier than working with Groovy, and we still have inconsistency as part of out build system is written in Kotlin (buildSrc and files like tokens.gradle.kts), and some in Groovy.

Reply:
I'd say we should look at how our gradle setup is in general - not just if it's using groovy or kotlin, but right now the compose module is the only build.gradle file that is using kotlin and it's confusing and makes changes harder as you need to change the syntax depending on which module you're in. The kotlin syntax also is more complex right now due to the way the dependencies are set up. So I'd rather change it to the shared system for now and then look at how we want the set up to be.
`buildSrc` + the token files being kotlin is less of an issue as the content is different - in this case the content is the same across different modules, but a different language is used, which causes a mental overhead.

Reply:
It's a good point – still I'm not sure we should rewrite one Gradle config to another without having the project defined and discussed, so we have decisions made and aligned.